CONTRIBUTORS.md

```diff
@@ -2,6 +2,9 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
 
 ## [Corporate CLA](Documentation/Contributors/CLAs/corporate-cla-agi-v1.0.txt)
 
+* [Bentley Systems ](https://www.bentley.com/)
+    * [Jason Crow](https://github.com/jason-crow)
```

Comment:
There's already a section for `Bentley Systems` at line 84.  Add yourself under `Paul Connelly` instead